en/02_Developer_Guides/09_Security/05_Secure_Coding.md

```diff
@@ -846,39 +848,10 @@ You can configure that by setting the following environment variables:
 
 ## Secure sessions and cookies
 
-We also want to ensure cookies are not shared between secure and non-secure sessions, so we must tell Silverstripe CMS to
-use a [secure session](/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie).
-To do this, you may set the `cookie_secure` parameter to `true` in your `config.yml` for `Session`.
-
-It is also a good idea to set the `samesite` attribute for the session cookie to `Strict` unless you have a specific use case for
-sharing the session cookie across domains.
-
-```yml
-SilverStripe\Control\Session:
-  cookie_samesite: 'Strict'
-  cookie_secure: true
-```
```

Comment:
This part should have been updated as part of https://github.com/silverstripe/developer-docs/pull/712.
I've opted to remove it, and instead just make sure this section links to the sessions docs, which includes this information.